sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_get_detailed_diagnostics_information_async.py

```diff
@@ -45,12 +48,12 @@ async def get_detailed_diagnostics_information_async(self):
         ]
 
         def callback(resp):
-            _LOGGER.info("document_count: {}".format(resp.statistics["document_count"]))
-            _LOGGER.info("valid_document_count: {}".format(resp.statistics["valid_document_count"]))
-            _LOGGER.info("erroneous_document_count: {}".format(resp.statistics["erroneous_document_count"]))
-            _LOGGER.info("transaction_count: {}".format(resp.statistics["transaction_count"]))
-            _LOGGER.info("model_version: {}".format(resp.model_version))
-            _LOGGER.info("raw_response: {}".format(resp.raw_response))
+            _LOGGER.debug("document_count: {}".format(resp.statistics["document_count"]))
+            _LOGGER.debug("valid_document_count: {}".format(resp.statistics["valid_document_count"]))
+            _LOGGER.debug("erroneous_document_count: {}".format(resp.statistics["erroneous_document_count"]))
+            _LOGGER.debug("transaction_count: {}".format(resp.statistics["transaction_count"]))
+            _LOGGER.debug("model_version: {}".format(resp.model_version))
+            _LOGGER.debug("raw_response in json format: {}".format(json.dumps(resp.raw_response)))
```

Comment:
Can we retrieve the response out of the callback function so a user can use it below? I think it makes sense to log the other pieces, but if someone goes through the trouble of passing the response hook to get the JSON response, I imagine it's because they want to use it, not just print it.